LICENSE

```diff
@@ -1,7 +1,3 @@
-The MIT License (MIT)
-
-Copyright (c) 2017-2020 Uber Technologies Inc.
-
```

Comment:
The LICENSE file is also used by the go generate command to generate mock files, will this change affect that?

Reply:
good call I will check - thanks